wordpress-mobile/WordPress-Android
# Prevent LoginEmailFragment from trying to disable nonexistent button when using old layout

This PR fixes an issue introduced by #12415, in which if we tried to disable the `LoginEmailFragment` primary button while having the new layout disabled, the app would crash. 

While this is not a problem for WPAndroid, as that flag is always true for this project, it would be a problem for WCAndroid when trying to use the updated login library.

Note: This issue was found while reviewing the changes made by the Unified Login & Signup project in order to [merge them back to the login library](https://github.com/wordpress-mobile/WordPress-Login-Flow-Android/pull/41). This PR also makes a minor, unrelated modification to the signatures of some `LoginAnalyticsListener` methods that receive property maps, in order to make that merge a bit easier.

To test:

1. Disable the `UNIFIED_LOGIN_AVAILABLE` flag and run the app.
1. On the Prologue screen, tap **Log In**.
1. Verify that the app hasn't crashed.

PR submission checklist:
